### PR TITLE
Add more configuration bindings (bookmark list, document title, document editor)

### PR DESCRIPTION
**Pull request by skoric**
Adds more configuration options (see `pspdfkit-flutter/example/lib/main.dart` for example):

1) Enabling/disabling bookmark list (`enableBookmarkList`)

Screenshot 2019-07-12 at 13 46 54

2) Show/hide document title (`showDocumentTitle`)

Screenshot 2019-07-12 at 13 29 34

3) Enabling/disabling document editor (`enableDocumentEditor`)

Screenshot 2019-07-12 at 13 47 04

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -125,11 +125,14 @@ class _MyAppState extends State<MyApp> {
         androidShowAnnotationListAction: true,
         showPageNumberOverlay: false,
         showPageLabels: true,
+        showDocumentTitle: false,
         invertColors: false,
         grayScale: false,
         startPage: 2,
         enableAnnotationEditing: true,
         enableTextSelection: false,
+        enableBookmarkList: false,
```
Comment (davidschreiber) [inline example/lib/main.dart]: I'd say we should enable bookmark list and document editor in the example by default (i.e. set these configuration options in the example to `true`, to make these features easier discoverable by trial user).

Comment (skoric) [reply example/lib/main.dart]: I was thinking in terms of like: this is enabled by default (in the basic example as well), so the custom configuration shows how you can disable them. But if you think more components should be in that presentation, I can set it to true no problem.

File: android/src/main/java/com/pspdfkit/flutter/pspdfkit/ConfigurationAdapter.java [hunk 2]
```diff
@@ -53,11 +52,14 @@
     private static final String ANDROID_SHOW_ANNOTATION_LIST_ACTION = "showAnnotationListAction";
     private static final String SHOW_PAGE_NUMBER_OVERLAY = "showPageNumberOverlay";
     private static final String SHOW_PAGE_LABELS = "showPageLabels";
+    private static final String SHOW_DOCUMENT_TITLE="showDocumentTitle";
```
Comment (tomassurin) [inline android/src/main/java/com/pspdfkit/flutter/pspdfkit/ConfigurationAdapter.java]:
```suggestion
    private static final String SHOW_DOCUMENT_TITLE = "showDocumentTitle";
```

File: android/src/main/java/com/pspdfkit/flutter/pspdfkit/ConfigurationAdapter.java [hunk 3]
```diff
@@ -53,11 +52,14 @@
...
     private static final String INVERT_COLORS = "invertColors";
     private static final String GRAY_SCALE = "grayScale";
     private static final String START_PAGE = "startPage";
     private static final String ENABLE_ANNOTATION_EDITING = "enableAnnotationEditing";
     private static final String ENABLE_TEXT_SELECTION = "enableTextSelection";
+    private static final String ENABLE_BOOKMARK_LIST="enableBookmarkList";
```
Comment (tomassurin) [inline android/src/main/java/com/pspdfkit/flutter/pspdfkit/ConfigurationAdapter.java]:
```suggestion
    private static final String ENABLE_BOOKMARK_LIST = "enableBookmarkList";
```

File: android/src/main/java/com/pspdfkit/flutter/pspdfkit/ConfigurationAdapter.java [hunk 4]
```diff
@@ -53,11 +52,14 @@
...
+    private static final String ENABLE_DOCUMENT_EDITOR ="enableDocumentEditor";
```
Comment (tomassurin) [inline android/src/main/java/com/pspdfkit/flutter/pspdfkit/ConfigurationAdapter.java]:
```suggestion
    private static final String ENABLE_DOCUMENT_EDITOR = "enableDocumentEditor";
```

File: example/lib/main.dart (hunk 1 above)
Comment (radazzouz) [reply example/lib/main.dart]: I also think that the default should be `true`, but for this example, I think it's okay to set it to false.

File: lib/configuration_options.dart [hunk 5]
```diff
@@ -60,9 +60,9 @@ const String enableAnnotationEditing = "enableAnnotationEditing";
 
 const String enableTextSelection = "enableTextSelection";
 
-const String enableBookmarkList = "enableBookmarkList";
+const String androidEnableBookmarkList = "androidEnableBookmarkList";
```
Comment (radazzouz) [inline lib/configuration_options.dart]: Shouldn't this be `const String androidEnableBookmarkList = "enableBookmarkList";` and `const String androidEnableDocumentEditor = "enableDocumentEditor";`?

Comment (skoric) [reply lib/configuration_options.dart]: It's just the string used as a key in configuration map. It matches `ANDROID_ENABLE_BOOKMARK_LIST` in `ConfigurationAdapter`.

Comment (radazzouz) [reply lib/configuration_options.dart]: Ah ok! Thanks! I thought it should be the same as https://github.com/PSPDFKit/pspdfkit-flutter/pull/21/files#diff-1bce22d614b794c1bd9d086607a85820R67.

**Pull request merged**